packages/fuchsia_ctl/test/command_line_test.dart

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 The Flutter Authors. All rights reserved.
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
-
+// @dart = 2.4
```

Comment:
Do you actually still need these opt-outs? I would have expected that upgrading package:test is enough...

Reply:
It was the first thing I tried, it didn't fix the issues without the upgrade.  I just left them in without trying to remove it.